src/Runner/Interpreters/ZendPhpCliInterpreter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * This file is part of the Nette Tester.
+ * Copyright (c) 2009 David Grudl (https://davidgrudl.com)
+ */
+
+namespace Tester\Runner\Interpreters;
+
+
+/**
+ * Zend PHP CLI executable.
+ */
+class ZendPhpCliInterpreter extends AbstractInterpreter
+{
+
+	public function __construct($path, $args = NULL)
+	{
+		parent::__construct($path, ' -n' . $args);
```

Comment:
I tried to normalized it across the Tester. The space was always [required](https://github.com/nette/tester/blob/5c4f5ec69d6894cd92bbeb1a216a63914f798761/src/Runner/ZendPhpInterpreter.php#L58). Now it is consistent.

Imho, it is not so important. Interpreters are "internal" classes.